engine/core/Router.php

```diff
@@ -5,7 +5,7 @@ class Router extends Singleton
     private array $dynamicSegments;
     private int $level = 0;
 
-    private function render($path_to_include = "pages/404.php")
+    private function render($path_to_include = "pages/404.php", $clean = false)
```

Comment:
My bad for not removing this completely: it is not used anymore 😊 you can revert it back!

Reply:
Okay !